lib/league_stats.rb

```diff
@@ -83,8 +83,8 @@ def worst_fans
 
 # Helper Methods
   def find_name(id)
-    team = @teams.find { |team| team.team_id == id }
-    team.teamname
+    team_id = @teams.find { |team| team.team_id == id }
+    team_id.teamname
```

Comment:
Very clear! Thanks for making this more specific.

Reply:
☝️